druntime/src/core/internal/vararg/aarch64.d

```diff
@@ -36,7 +36,14 @@ extern (C++, std) struct __va_list
 }
 
 ///
-alias va_list = __va_list;
+alias __va_list_tag = __va_list;
+
+version (DigitalMars)
+    ///
+    alias va_list = __va_list*;
+else
+    ///
+    alias va_list = __va_list; // kludge - va_list always passed by ref (!)
```

Comment:
The kludge is the DMD branch.

Reply:
May I quote you: "Yeah I don't know what they were thinking..." and you added special case code for it :-)

Reply:
Oh well, I guess the mangling is the smallest concern here. I forgot that you can have a `va_list` local too, not just params...

Reply:
But no more objections from me, it's `version (DigitalMars)` now.